From 86b758e747946437885526ca93efd2072c0af69a Mon Sep 17 00:00:00 2001
From: John Lees <lees.john6@gmail.com>
Date: Fri, 22 Dec 2023 13:23:16 +0000
Subject: [PATCH] Dynamic link cuda rt

---
 src/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 4ce170d..f2defc9 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -14,7 +14,7 @@ LDFLAGS=-Wl,-as-needed
 
 CPPFLAGS+=-I"." -I$(LIBLOC)/include -I$(LIBLOC)/include/boost
 LDFLAGS+= -L$(LIBLOC)/lib
-CUDA_LDLIBS=-lcudadevrt -lcudart_static $(LDLIBS)
+CUDA_LDLIBS=-lcudadevrt $(LDLIBS)
 # python specific options
 CPPFLAGS += -DPYTHON_EXT -DNDEBUG -DSCE_EXPORTS $(shell python3 -m pybind11 --includes)
 PYTHON_LIB = SCE$(shell python3-config --extension-suffix)
@@ -22,10 +22,10 @@ PYTHON_LIB = SCE$(shell python3-config --extension-suffix)
 ifdef GENARCH
 	GENCODE=${GENARCH}
 else
-	GENCODE=-gencode arch=compute_70,code=sm_70 -gencode arch=compute_75,code=sm_75  -gencode arch=compute_80,code=sm_80 -gencode arch=compute_86,code=sm_86
+	GENCODE=-gencode arch= -gencode arch=compute_80,code=sm_80 -gencode arch=compute_86,code=sm_86
 endif
 CUDA_LDFLAGS =-L$(LIBLOC)/lib -L${CUDA_HOME}/targets/x86_64-linux/lib/stubs -L${CUDA_HOME}/targets/x86_64-linux/lib
-CUDAFLAGS +=-Xcompiler -fPIC -Xptxas -dlcm=ca --cudart static --relocatable-device-code=true $(GENCODE)
+CUDAFLAGS +=-Xcompiler -fPIC -Xptxas -dlcm=ca --relocatable-device-code=true $(GENCODE)
 
 CPU_OBJS=wtsne_cpu.o python_bindings.o
 GPU_OBJS=wtsne_gpu.cu.o
-- 
2.39.3 (Apple Git-145)

