From 614305a87c21a3a7b749c709a956e6d19f925e1d Mon Sep 17 00:00:00 2001
From: anmol thapar <mr.anmolthapar@gmail.com>
Date: Tue, 11 Mar 2025 09:29:44 +0000
Subject: [PATCH] Remove CONFIG option from Boost package requirement in
 CMakeLists.txt

---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1320976..c210b60 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33,9 +33,8 @@ else()
 endif()
 
 # Add libraries
-find_package(Python COMPONENTS Interpreter Development REQUIRED)
 find_package(ZLIB REQUIRED)
-find_package(Boost CONFIG REQUIRED)
+find_package(Boost REQUIRED)
 find_package(pybind11 REQUIRED)
 include_directories(${Boost_INCLUDE_DIRS})
 
-- 
2.34.1

